_articles/de/best-practices.md

```diff
@@ -1,283 +1,319 @@
+Dinge aufzuschreiben macht es einfacher, nein zu sagen, wenn etwas nicht in den Anwendungsbereich passt. Es macht es auch einfacher für die Menschen, mitzumachen und zu helfen. Sie wissen nie, wer Ihr Projekt lesen oder nutzen könnte.
 
-Even if you don't use full paragraphs, jotting down bullet points is better than not writing at all.
+Selbst wenn Sie keine vollständigen Absätze niederschreiben, ist es besser, Stichworte aufzulisten, als gar nicht zu schreiben.
 
-Remember to keep your documentation up-to-date. If you're not able to always do this, delete your outdated documentation or indicate it is outdated so contributors know updates are welcome.
+Denken Sie daran, Ihre Dokumentation auf dem neuesten Stand zu halten. Wenn Sie dies nicht immer tun können, löschen Sie Veraltetes oder markieren es als solches, damit die Mitwirkenden wissen, dass Updates gerne angenommen werden.
 
-### Write down your project's vision
+### Schreiben Sie Ihre Projektsvision auf
 
-Start by writing down the goals of your project. Add them to your README, or create a separate file called VISION. If there are other artifacts that could help, like a project roadmap, make those public as well.
+Schreiben Sie zunächst die Ziele Ihres Projekts auf. Fügen Sie sie Ihrer README hinzu oder erstellen Sie eine separate Datei namens VISION. Wenn es andere dafür nützliche Artefakte gibt (z.B. eine Projekt-Roadmap) machen Sie diese auch öffentlich.
 
-Having a clear, documented vision keeps you focused and helps you avoid "scope creep" from others' contributions.
+Eine klare, dokumentierte Vision hilft Ihnen, sich zu konzentrieren und das "Scope Creep" von Beiträgen anderer zu vermeiden.
 
-For example, @lord discovered that having a project vision helped him figure out which requests to spend time on. As a new maintainer, he regretted not sticking to his project's scope when he got his first feature request for [Slate](https://github.com/lord/slate).
+Zum Beispiel entdeckte @lord, dass ihm eine Projektvision herauszufinden half, auf welche Anfragen er Zeit verwenden sollte. Als frisch gebackener Maintainer bedauerte er, dass er sich nicht auf den Kern seines Projekts fokussiert hat, als er seine erste Feature-Anfrage für [Slate](https://github.com/lord/slate) erhielt.
 
 <aside markdown="1" class="pquote">
   <img src="https://avatars.githubusercontent.com/lord?s=180" class="pquote-avatar" alt="avatar">
-  I fumbled it. I didn't put in the effort to come up with a complete solution. Instead of an half-assed solution, I wish I had said "I don't have time for this right now, but I'll add it to the long term nice-to-have list."
+
+  Ich habe es zurechtgefummelt. Ich habe mir nicht die Mühe gemacht, eine komplette Lösung zu finden. Statt einer halbherzigen Lösung wünschte ich, ich hätte gesagt: "Ich habe jetzt keine Zeit dafür, aber ich werde es auf die lange Nice-to-have-Liste setzen."
```

Comment:
`Ich habe es mir zurechtgefummelt. `
`auf die lange Nice-to-have-Liste setzen.` -> `auf die langfristige Nice-to-have-Liste setzen.`